src/utils/formatters.js

```diff
@@ -199,6 +199,18 @@ export const collapseChoices = (choices, depth) => {
   return result.flat(100);
 };
 
+export const calculatePrettyTimeElapsedSince = createdDate => {
+  const now = new Date();
+  const formattedDistance = formatDistance(
```

Comment:
with these `date-fns` functions it's important to wrap in a try/catch block. i have definitely experienced them crashing the whole page when you provide a bad value to them.